src/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use '*' selector (it impacts performance). Set styles only for elements that require them.

Reply:
+- justo